src/Microsoft.PowerApps.TestEngine.Tests/ConsoleOutputTests.cs

```diff
@@ -139,5 +139,41 @@ public void TestTestCaseEndFailed()
             // Assert that the expected output matches the console output of the function
             Assert.Contains(expected, printer.ToString());
         }
+
+        [Fact]
+        public void TestMultipleBrowserRuns()
+        {
+            var printer = new StringWriter();
+
+            // Set output
+            Console.SetOut(printer);
+
+            // Run tests on first Browser
+            _testEngineEventHandler.SetAndInitializeCounters(2);
+            _testEngineEventHandler.SuiteBegin("TestSuiteName", "./testDirectory", "Chromium", "make.powerapps.com/testapp&source=testengine");
+            _testEngineEventHandler.TestCaseBegin("Case1");
+            _testEngineEventHandler.TestCaseEnd(true);
+            _testEngineEventHandler.TestCaseBegin("Case2");
+            _testEngineEventHandler.TestCaseEnd(false);
+            _testEngineEventHandler.SuiteEnd();
+
+            // Run tests on second Browser
+            _testEngineEventHandler.SetAndInitializeCounters(2);
+            _testEngineEventHandler.SuiteBegin("TestSuiteName", "./testDirectory", "Firefox", "make.powerapps.com/testapp&source=testengine");
+            _testEngineEventHandler.TestCaseBegin("Case1");
+            _testEngineEventHandler.TestCaseEnd(true);
+            _testEngineEventHandler.TestCaseBegin("Case2");
+            _testEngineEventHandler.TestCaseEnd(false);
+            _testEngineEventHandler.SuiteEnd();
+
+            // Assert that the expected console output matches 
+            Assert.Contains("\nTest suite summary", printer.ToString());
+            Assert.Contains("Total cases: 2", printer.ToString());
```

Comment:
While there are 2 cases, there was 4 executions of cases. It might be weird to see "2" cases and "4" failures, wouldn't it? How does Karma handle this sort of thing when a single test case is run multiple times in a run?

Reply:
We handle console output separately for each test run. So the output would look like this in the scenario 
``` 
Running test suite: TestSuiteName
   Test results will be stored in: ./testDirectory
   Browser: Chromium
   App URL: make.powerapps.com/testapp
Test case: Case1
   Result: Passed
Test case: Case2
   Result: Failed

Test suite summary
Total cases: 2
Cases passed: 1
Cases failed: 1
Running test suite: TestSuiteName
   Test results will be stored in: ./testDirectory
   Browser: Firefox
   App URL: make.powerapps.com/testapp
Test case: Case1
   Result: Passed
Test case: Case2
   Result: Failed

Test suite summary
Total cases: 2
Cases passed: 1
Cases failed: 1